### PR TITLE
Update authentication docs

### PR DESCRIPTION
**Pull request by loay**
connect to https://github.com/strongloop/loopback-next/issues/260
cc @bajtos @raymondfeng @ritch @superkhau

### CODE REVIEW

File: packages/authentication/README.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
 A LoopBack component for authentication support.
+
+# Overview
+ It demonstrates how to use LoopBack's user models and passport to interact with other authentication providers.
+ User can login using a passport.js strategy, which could include a third party provider.
+
+
+# Installation
+
+Then in your Node.js project root, run:
```
Comment (bajtos) [inline packages/authentication/README.md]: I don't think we are using a sentence like this in other places. See https://loopback.io/doc/en/contrib/README-guidelines.html#installation

File: packages/authentication/README.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+
+```shell
+npm install -S @loopback/authentication
```
Comment (bajtos) [inline packages/authentication/README.md]: IIRC, `-S` is for `--save-dev`, that's not right. This component will be typically used in production, also not many people are familiar with `-s` vs. `-S`. Let's use full `--save` please.

**Comment by loay** [comment]
@bajtos PTAL again. Thanks.

File: packages/authentication/README.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+
+```shell
+npm install --save @loopback/authentication
+```
+
+# Basic use
+
+
+ ```ts
+
```
Comment (bajtos) [inline packages/authentication/README.md]: Please remove extra empty line.

File: packages/authentication/README.md [hunk 4]
```diff
@@ -1,3 +1,47 @@
...
+   const strategy = new BasicStrategy(async (username, password) => {
+     return await findUser(username, password);
+   };
+   getAuthenticatedUser(strategy, ParsedRequest);
```
Comment (bajtos) [inline packages/authentication/README.md]: Why are all lines indented by two spaces? Can you remove this extra indentation please?

File: packages/authentication/README.md [hunk 5]
```diff
@@ -1,3 +1,47 @@
...
+   ```
```
Comment (bajtos) [inline packages/authentication/README.md]: ditto - the triple-backtick sequence should not be indented.

**Comment by bajtos** [comment]
> Please ask @crandmck or @superkhau for the final approval.

Ignore my comment, I see that Rand has already approved 👍 

Please fix the indentation, rebase on top of the latest master, and merge :shipit:

**Pull request merged**